pages/shields/oled.rst

```diff
@@ -0,0 +1,139 @@
+OLED Shield
+***********
+
+.. highlight:: python
+   :linenothreshold: 4
+
+.. figure:: /images/oled_shield_top.png
+   :width: 270
+
+   OLED Shield
+
+The `OLED Shield`_ is a Wemos D1 Mini shield that hosts a *tiny* 17mm 64x48
+monochrome OLED display.
+
+.. NOTE::
+
+    What is an *OLED*? It stand for Organic Light-Emitting Diode. It describes
+    the technology used to turn on or off pixels - dots - on the display. The
+    term is largely unimportant, just think of it like a tiny little TV where
+    you can control every pixel.
+
+Although small, this shield packs a punch! You can draw pixels, render lines
+and display text.
```

Comment:
This description seems limiting - with enough effort you can display anything that can be represented in 64x48 colour pixels